### DISCUSSION TITLE
How to access NS runtime from iOS or Android

### DISCUSSION THREAD

**Discussion by Kampii** [NativeScript/NativeScript#10137] [Q&A]
It seems that NS version 4 or 5 allowed this. A lot has changed now in v8 and I am not able to find how to do it nor any documentation. Has anyone succeeded with this or can share any pointers?

**Comment by farfromrefug** [comment]
@Kampii what do you mean by accessing the runtime?

**Reply by Kampii** [reply]
@farfromrefug Referring to Nativescript runtime which we can use to call ns functions from ios

**Reply by farfromrefug** [reply]
@Kampii you want to call JS functions from objective c?
The ususal way is to use delegate which are set from JS. When the objective c delegate is call then it calls the JS implementation of it

**Reply by Kampii** [reply]
@farfromrefug Ah, let us verify this, thank you so much for the pointer! I hope to update this thread asap.

**Comment by Adnaan03** [comment]
@farfromrefug we want to start NS app on button click from another scene, after tns prepare ios we added new scene which have a button and on click of that button we want to start NS app but we are not able to do it
We have acheived the same on android

**Reply by farfromrefug** [reply]
@Adnaan03 @NathanWalker might know to do that. Never did that myself
